app/presenters/organisation_presenter.rb

```diff
@@ -17,10 +17,15 @@ def organisation_type
     I18n.t("organisation.organisation_type.#{super}")
   end
 
-  def filename_for_activities_template
+  def filename_for_activities_template(type:)
+    human_readable_type = {
+      ispf_oda: "ISPF_ODA",
+      non_ispf: "GCRF_NF_OODA"
+    }.fetch(type, "").tr(" ", "_")
```

Comment:
I've simplified it so we aren't adding a fallback, since we don't have a clear use case for needing to fall back as of yet. Currently it won't error - I moved the method for this formatting back into the importer (in anticipation of reusing it for levels C and D_, and by default it returns `nil` if the type doesn't exist, so this method will just have no type interpolated. Perhaps we should raise an error though